nuxeo-arender-previewer-apb/apb.yml

```diff
@@ -30,6 +30,11 @@ plans:
       type: string
       title: Url of the Nuxeo client
       display_group: Client Nuxeo
+    - name: previewer_nuxeo_client_timeout
```

Comment:
Du coup il faut aussi ajouter la ligne suivante dans `defaults/main.yaml`:

```
previewer_nuxeo_client_timeout: 60
```
et lancer un `make build` pour que le fichier `Dockerfile` soit modifié.

Reply:
This is required for every parameter that we add to the `apb.yaml`

Reply:
Done but cannot run (as of today the make build command , software requirements issues)